src/trio/_tools/gen_exports.py

```diff
@@ -3,6 +3,9 @@
 Code generation script for class methods
 to be exported as public API
 """
+
+# mypy: disable-error-code="unused-ignore"
```

Comment:
Is mypy saying that the `astor` call's `# type: ignore` is unused? Why/why is it wrong?

Reply:
Mypy doesn't care about untyped libraries; pyright strict does.

Reply:
I would much rather type ignore the specific lines that are causing issues than the entire file, because if something in our assumptions change later that will make debugging more tricky, and plus I would consider it better practice.